.github/workflows/examples-nrfconnect.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: connectedhomeip/chip-build-nrf-platform:0.7.3
+            image: connectedhomeip/chip-build-nrf-platform:0.7.16
```

Comment:
you must change runners in integrations/cloudbuild as well, since those run nrf compilations as well.

In those files, please change all instances not just NRF, to avoid downloading 20+GB images several times.

Reply:
Ok, applied, please re-review.